### ISSUE TITLE
[4.1] Missing WCAG standards for keyboard accessibility

### ISSUE THREAD

**Issue by Krshivam25** [joomla-projects/gsoc21_accessibility#4]
### What needs to be fixed: 
![Screenshot (286)_LI](https://user-images.githubusercontent.com/49578630/121768646-e5fedb00-cb7c-11eb-9dbe-be88796d3252.jpg)
